### ISSUE TITLE
Tebex for credits.

### ISSUE THREAD

**Issue by fifo8** [Minehut/Meta#824]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**
Ingame/website

**Describe the feature**
A plugin or something like that, that hooks up to your minehut account, and when a person purchases something in-game instead of it sending money to your Paypal it sends credits straight to your account. (this wouldn't require you to have a Paypal or card as people who will use this most likely won't have one).

**How the feature is useful**
It will benefit people who don't have Paypal or a card to upgrade their server to have better hardware or a higher player limit

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by erenkarakal** [comment]
i dont think this is ever going to happen as tebex and minehut has to partner for that

**Comment by TrueReassembly** [comment]
No. Just no. If you actually get the money from servers, you can spend it on both credits and other stuff like builds / personal stuff, limiting it to credits would just sorta lower incentive for it.

**Comment by fifo8** [comment]
> No. Just no. If you actually get the money from servers, you can spend it on both credits and other stuff like builds / personal stuff, limiting it to credits would just sorta lower incentive for it.

no like make it an option not an obligation so you could still use tebex this is just an option for people who don't have a card

**Comment by fifo8** [comment]
> it would be possible for them to make a tebex **like** plugin that minehut just got the money and you got the equivalent in credits but that is a very big suggestion

yea this is exactly what I intended to ask

**Comment by ahoulton** [comment]
Hello there @fifo8 and @Zac694, jumping in.

> no like make it an option not an obligation so you could still use tebex this is just an option for people who don't have a card

Pretty sure you can use PayPal without a card if the account has a balance?

> it would be possible for them to make a tebex **like** plugin that minehut just got the money and you got the equivalent in credits but that is a very big suggestion

Hi there, things like this are harder to make essentially not worth the developers time. While I do understand where you are going with this, but it's easier to say then to do.

**Comment by fifo8** [comment]
> Hello there @fifo8 and @Zac694, jumping in.
> 
> > no like make it an option not an obligation so you could still use tebex this is just an option for people who don't have a card
> 
> Pretty sure you can use PayPal without a card if the account has a balance?
> 
> > it would be possible for them to make a tebex **like** plugin that minehut just got the money and you got the equivalent in credits but that is a very big suggestion
> 
> Hi there, things like this are harder to make essentially not worth the developers time. While I do understand where you are going with this, but it's easier to say then to do.

i can use Paypal without a card but minehut needs a card on the paypal

**Comment by DaRealMinecraftexpert** [comment]
Some don't have an account on paypal
